README.md

```diff
@@ -15,6 +15,39 @@ for Windows' [Google Group](http://groups.google.com/group/git-for-windows),
 and [contribute bug
 fixes](https://github.com/git-for-windows/git/wiki/How-to-participate).
 
+To build Git for Windows, please either install [Git for Windows'
+SDK](https://gitforwindows.org/#download-sdk), start its `git-bash.exe`, `cd`
+to your Git worktree and run `make`, or open the Git worktree as a folder in
+Visual Studio.
+
+To verify that your build works, use one of the following methods:
+
+- If you want to test the built executables within Git for Windows' SDK,
+  prepend `<worktree>/bin-wrappers` to the `PATH`.
+- Alternatively, run `make install` in the Git worktree.
+- If you need to test this in a full installer, run `sdk build
+  git-and-installer`.
+- You can also "install" Git into an existing portable Git via `make install
+  DESTDIR=<dir>` where `<dir>` refers to the top-level directory of the
+  portable Git.
```

Comment:
I've never used the portable version, so I'll play dumb here.  So this will overwrite the EXEs in the portable Git installation.  We should advise that they confirm their PATH is correct??

Reply:
So for example, their PATH should be prefixed with `<dir>/bin`, right?

Reply:
You mean the `<dir>` should be correct? Yes, that would be helpful... 😁

Reply:
> So for example, their PATH should be prefixed with `<dir>/bin`, right?

I am not sure what this suggestion means. Is this about calling portable Git? That should be done via launching `git-bash.exe` or `git-cmd.exe` in the unpacked top-level directory of the portable Git...

Reply:
I'm not sure I know what I'm talking about. :smile:

I was just thinking that when we do "make prefix=/c/foo install" we need to 
prepend "/c/foo/bin" to the PATH (and not just "/c/foo").  I assumed there is
a similar difference for the portable version, but maybe not.

Reply:
Oh, I think I understand now... I thought it was common knowledge that you need to add a portable Git's `/cmd` directory to the `PATH`... Will fix.